src/routers/handlers/payment_callback/payment.callback.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from "express";
+import { GenericHandler } from "./../generic";
+import { createAndLogError, logger } from "../../../utils/logger";
+import { PrefixedUrls } from "../../../utils/constants/urls";
+import { must } from "../../../utils/session";
+import { getAccountsFilingId } from "../../../utils/session";
+import { Session } from "@companieshouse/node-session-handler";
+
+export class PaymentCallbackHandler extends GenericHandler {
+    constructor() {
+        super({
+            title: "Payment Callback – GOV.UK",
+            backURL: null,
+        });
+    }
+
+    async executeGet(
+      req: Request,
+      _response: Response) {
+        const session = req.session as Session;
+        const paymentStatus = req.query.status;
+        const accountsFilingId = must(getAccountsFilingId(req.session));
```

Comment:
Do we need the accounts filing id, if it's only for the log message, the log messages context id should be able to identify it